mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/MailboxMapper.java

```diff
@@ -99,4 +100,34 @@ boolean hasChildren(Mailbox mailbox, char delimiter)
      * @throws MailboxException 
      */
     List<Mailbox> list() throws MailboxException;
+
+    /**
+     * Return the list of <code>MailboxAnnotation</code> based on the selected mailbox
+     * 
+     * @param mailbox
+     * @return List<MailboxAnnotation>
+     * @throws MailboxException
```

Comment:
the 3 methods needs more documentation (probably very similar to the upper layer)